libs/product/driver/shopify/src/product.service.spec.ts

```diff
@@ -139,20 +149,26 @@ describe('Driver | Shopify | Product | ProductService', () => {
 
       op.flush({
         data: {
-          handle: '',
-          id: product.id,
-          title: product.name,
-          description: product.description,
-          availableForSale: product.in_stock,
-          onlineStoreUrl: product.canonicalUrl,
-          priceRange: {
-            maxVariantPrice: {
-              amount: product.price,
-              currencyCode: 'USD',
+          product: {
+            handle: '',
+            id: product.id,
+            title: product.name,
+            description: product.description,
+            availableForSale: product.in_stock,
+            onlineStoreUrl: product.canonicalUrl,
+            priceRange: {
+              minVariantPrice: {
+                amount: product.price,
+                currencyCode: 'USD',
+              },
+              maxVariantPrice: {
+                amount: product.price,
+                currencyCode: 'USD',
+              },
+            },
+            images: {
+              nodes: [],
```

Comment:
We should make factories for this entity, it doesn't need to block this PR though

Reply:
100% agree.